src/write.c

```diff
@@ -1119,7 +1178,11 @@ static avifResult avifEncoderAddImageItems(avifEncoder * encoder,
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
                             : (itemCategory == AVIF_ITEM_GAIN_MAP) ? infeNameGainMap
 #endif
-                                                                   : infeNameColor;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+                            : (itemCategory >= AVIF_SAMPLE_TRANSFORM_MIN_CATEGORY && itemCategory <= AVIF_SAMPLE_TRANSFORM_MAX_CATEGORY)
+                                ? infeNameSampleTransform
+#endif
+                                : infeNameColor;
```

Comment:
The cascaded ternary `? : `operators are difficult to parse. This would be clearer if rewritten as if-else-if statements.